core/src/main/java/hudson/model/Fingerprint.java

```diff
@@ -1316,6 +1316,10 @@ public Api getApi() {
      * Loads a {@link Fingerprint} from the Storage with the given unique id.
      * @return Loaded {@link Fingerprint}. {@code null} if the config file does not exist or
      * malformed.
+     *
+     * In case an external storage is configured on top of a file system based storage, first external storage is
```

Comment:
```suggestion
     * In case an external storage is configured on top of a file system based storage: 

     1) External storage is...
     2) If not found and fingerprint is present...
```

This description is difficult to read, see above suggestion on how to refactor it (basically don't have one long run-on sentence here :))